mRemoteV1/Config/DatabaseConnectors/SqlDatabaseConnectionTester.cs

```diff
@@ -7,18 +7,19 @@ namespace mRemoteNG.Config.DatabaseConnectors
     /// <summary>
     /// A helper class for testing database connectivity
     /// </summary>
-    public class SqlDatabaseConnectionTester
+    public class DatabaseConnectionTester
```

Comment:
Since the class was renamed, the file _should_ be renamed as well.

This is best done via the git command line - in order to keep all history:
git mv SqlDatabaseConnectionTester.cs DatabaseConnectionTester.cs

If you're not familiar or comfortable with the git command line, let me know and I should be able to push the necessary change directly into this PR.

Reply:
Good catch, I suspect it just slipped my mind, and VS didn't prompt me to do so. will fix.

Reply:
Done - pushed a rebase to include this in the main commit